lib/receipt.rb

```diff
@@ -0,0 +1,41 @@
+class Receipt
+  attr_accessor :receipt, :business, :address, :phone, :line_items, :tax, :total
+
+  def initialize
+    @receipt = []
+    @business = ''
+    @address = ''
+    @phone = ''
+    @line_items = []
+    @tax = ''
+    @total = ''
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.